torch/onnx/_exporter_states.py

```diff
@@ -25,6 +26,12 @@ class SymbolicContext:
         onnx_block (_C.Block): Current ONNX block that converted nodes are being appended to.
     """
 
+    @_deprecation.deprecated(
+        "1.13",
+        "TBD",
```

Comment:
Forgot to mention, do you intend to keep TBD?

Reply:
Updated to 1.14

Reply:
README updated. Public user doc to be updated in another PR.